testutil/simapp/simapp.go

```diff
@@ -20,10 +21,13 @@ func New(dir string) *app.App {
 
 	a := app.New(logger, db, nil, true, simtestutil.EmptyAppOptions{})
 	// InitChain updates deliverState which is required when app.NewContext is called
-	a.InitChain(&abci.RequestInitChain{
+	_, err := a.InitChain(&abci.RequestInitChain{
 		ConsensusParams: defaultConsensusParams,
 		AppStateBytes:   []byte("{}"),
 	})
+	if err != nil {
+		return nil
```

Comment:
Should we panic here?

Reply:
Returning nil will eventually lead to a panic depending where it is used but to be more straighforward, added a panic there